polyfills.js

```diff
@@ -19,6 +19,7 @@ process.chdir = function(d) {
   cwd = null
   chdir.call(process, d)
 }
+process.chdir.__proto__ = chdir
```

Comment:
```suggestion
if (Object.setPrototypeOf) Object.setPrototypeOf(process.chdir, chdir)
```

Per #201 we'll need to use `Object.setPrototypeOf` when it is available.